myLedger_data/myLedger_inputs.txt

```diff
@@ -1 +1,2 @@
-Bd/loanv/10000.0t/2023-01-01p/Nonen/posb banko/2023-05-06
+Acadd/hiv/10.1t/2000-10-10p/Nonen/Noneo/None
+Fd/Kappav/11.11t/2000-11-11p/Nonen/Noneo/None
```

Comment:
There might not be a need to push the txt file in future updates as the list should depend on user inputs and not predetermined ones